docs/pbc/all/cart-and-checkout/latest/cart-page-performance-configuration.md

```diff
@@ -132,3 +134,98 @@ Update product relation widgets on the cart page to use carousel rendering to im
 
 {% endraw %}
 Verify that these widgets are present in your project Twig templates, and update them as needed.
+
+# Cart operations performance configuration
+
+To optimize the performance of cart operations such as adding, removing, or updating items in the cart, consider the following configurations:
+
+## 1. Prerequisites
+
+Ensure that you have the latest versions of the following packages installed in your Spryker project:
+
+```php
+composer require spryker/merchant:"^3.19.0" \
+    spryker/persistent-cart:"^3.14.0" \
+```
+
+## 2. Configurations
+
+You can divide cart operations into two categories:
+
+ - **Inside-cart operations** (add to cart, remove from cart, change item quantity, and similar actions)
```

Comment:
Can we name this Cart items operations?
```suggestion
 - **Cart items operations** (add to cart, remove from cart, change item quantity, and similar actions)
```

Reply:
I think Inside-cart operations describe it better